docs/references/pysdk_api_reference.md

```diff
@@ -1351,12 +1247,12 @@ Deletes rows from the table based on the specified condition.
 
 ### Parameters
 
-#### cond: `str` (not empty), *Optional*
+#### cond: `str` (non-empty), *Optional*
 
 A condition or filter that determines which rows to delete from the table. The parameter can be an expression, a function, or any other form of conditional logic that evaluates to `True` for the rows that should be deleted. If `cond` is not specified or set to `None`, the method will delete all rows in the table.
 
 :::tip NOTE
-- The `cond` parameter currently supports 'and' and 'or' logical expressions only. Additional expressions like 'between' and 'in' will be available in future updates.
+- The `cond` parameter currently supports 'and' and 'or' logical expressions only.
```

Comment:
need to double check

Reply:
BETWEEN AND and IN are not implemented yet.